src/types.hpp

```diff
@@ -74,4 +93,6 @@ static_assert(sizeof(VoxelColorId) == sizeof(MatColorId));
  **********/
 
 // size in pixels of graphics objects
-using screen_size_t = size_t;
+using screen_size_t = int;
```

Comment:
```suggestion
using screen_size_t = int64_t;
```

Really should be `ssize_t` with a compile check to define it if its not defined.

Reply:
nino plese read comments. Im going to change this ie (wrap get_window_size), but probally not in this pr.

Reply:
I made this comment BEFORE you updated the PR.